plugins/modules/openbsd_pkg.py

```diff
@@ -169,7 +169,11 @@ def get_package_state(names, pkg_spec, module):
         rc, stdout, stderr = execute_command(command, module)
 
         if stderr:
-            module.fail_json(msg="failed in get_package_state(): " + stderr)
+            match = "Can\'t find inst:%s" % (name) in stderr
```

Comment:
I think this way it's easier to understand what it does:
```suggestion
            match = ("Can't find inst:%s" % name) in stderr
```